src/Trixi.jl

```diff
@@ -257,10 +262,13 @@ function __init__()
     using .Plots: Plots
   end
 
-  @require Makie="ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a" begin
-    include("visualization/recipes_makie.jl")
-    using .Makie: Makie, GeometryBasics
-    export iplot, iplot! # interactive plot
+  # Until Julia v1.9 is the minimum required version for Trixi.jl, we still support Requires.jl
+  @static if VERSION < v"1.10.0-DEV.1288" && VERSION < v"1.9.1"
```

Comment:
```suggestion
  # We do not check `isdefined(Base, :get_extension)` since Julia v1.9.0
  # does not load package extensions when their dependency is loaded from
  # the main environment
  @static if !(VERSION >= v"1.9.1")
```